src/pkg/bundle/dev.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+// Package bundle contains functions for interacting with, managing and deploying UDS packages
+package bundle
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/defenseunicorns/uds-cli/src/config"
+	zarfCLI "github.com/defenseunicorns/zarf/src/cmd"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	zarfUtils "github.com/defenseunicorns/zarf/src/pkg/utils"
+)
+
+// CreateZarfPkg creates a zarf package if its missing when in dev mode
+func (b *Bundle) CreateZarfPkgs(srcDir string) {
+	path := filepath.Join(srcDir, b.cfg.CreateOpts.BundleFile)
+	if err := zarfUtils.ReadYaml(path, &b.bundle); err != nil {
+		message.Fatalf(err, "Failed to read bundle.yaml: %s", err.Error())
```

Comment:
Thinking out loud, since dev deploy uses the TUI for the deploy, as a follow up to this, we could add some extra TUI indicators this particular deploy operation

Reply:
created #540 for tracking